app/assets/stylesheets/components/_theme.scss

```diff
@@ -0,0 +1,71 @@
+$Theme-spacing-base: 18px;
+$Theme-spacing-small: 9px;
+
+$Theme-fontSize-base: 18px;
+$Theme-fontSize-large: 22px;
```

Comment:
You're right. I meant 18px instead of 16px. It's supposed to be your base font size